Tools/serial/generate_config.py

```diff
@@ -269,7 +269,7 @@ def parse_yaml_serial_config(yaml_config):
             else:
                 default_port_str = port_config['default']
 
-            if default_port_str != "":
+            if default_port_str != "" and default_port_str != 0:
```

Comment:
When is this a number?

Reply:
Daniel had used that as a "null" default for the sagetech transponder driver. However I see that just two days back he fixed that. I no longer need this, I'll remove it.